articles/virtual-machines/flash-azure-monitor.md

```diff
@@ -43,6 +43,17 @@ To navigate to the [metrics explorer](/azure/azure-monitor/essentials/metrics-ge
 
 For a description of the metric's display values, see [VM availability metric](monitor-vm-reference.md#vm-availability-metric-preview).
 
+Users can split the VM availability by the 'Context' property.
+   :::image type="content" source="media/flash/context-property-filter.png" alt-text="Screenshot of the newly added Context property of VM availability Metric on Azure portal." lightbox="media/flash/context-property-filter.png" :::
+
+Note: The context for migrations (Service Healing and Live Migration) is unknown by default. We are working on making this information available in the future.
+
+The legend at the bottom indicates the color associated with the possible Context: Unknown, Platform Initiated, and User Initiated.
+   :::image type="content" source="media/flash/split-by-context-property.png" alt-text="Screenshot of the VM availability Metric split by Context on Azure portal." lightbox="media/flash/split-by-context-property.png" :::
+
+Users can also create an alert rule based on dimension values. Under the Condition tab, choose VM Availability Metric as the Signal name. In the Split by dimensions section, enter Context as the dimension name and select the corresponding dimension values.
```

Comment:
Please write this as instructional information -- we need to use an "active" voice in our docs or they get graded low and over time will prevent you from being able to publish changes to this doc until it's addressed. If you look at the "Acrolinx" score card for this doc, it's already being docked points for this section/lines. 

Maybe change all of this to something like the following and make a new sub section using ###. Also, when referencing UI elements, please use italic (two stars on each side of text) for values you want selected/entered, and use bold (one star on each side of text) for UI elements like the names of sections/panels/tabs/etc. Use the exact names. 
 
#### Create a rule based on dimension values
Under the *Condition* tab, choose **VM Availability Metric** as the *Signal* name. In the *Split by dimensions* section, enter *Context* as the dimension name and select the corresponding dimension values.
<image from line 55 here>